Jenkinsfile-long

```diff
@@ -40,7 +40,7 @@ pipeline {
             steps {
                 timeout(time: 10, unit: 'MINUTES') {
                     sh './systemtests/scripts/setup-openshift.sh "systemtests"'
-                    sh 'sudo chmod -R 777 /var/lib/origin/openshift.local.config'
+                    sh 'sudo chmod -R 777 /var/lib/origin/ || true'
```

Comment:
How can this command fail?

Reply:
So openshift 3.9 is by default installed to /var/lib/origin, but oc 3.10 is installed into jenkins workspace, which is cool because after tests folder is cleaned. So I leave there chmod when job is running with 3.9.

Reply:
@lulf so when job is running with oc 3.10, /var/lib/origin foler does not exists